code/data_structures/src/tree/heap/binomial_heap/binomial_heap.java

```diff
@@ -0,0 +1,124 @@
+public class BinomialHeap{
+
+	private Node array[];//Heap's array
```

Comment:
Instead of comments for these things, why not just name the variable to what it is, i.e call this variable `heapArray`.